sys/include/net/gnrc/netdev/sx127x_adpt.h

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Yes it could be, I hesitated between the 2 and went for sx127x. But I agree lora.h could be better since it's quite generic.